Assets/XML/Text/Technologies.xml

```diff
@@ -382,15 +411,19 @@
 	<TEXT>
 		<Tag>TXT_KEY_TECH_GEOGRAPHY</Tag>
 		<English>Geography</English>
-		<French>Geography</French>
+		<French>
+			<Text>G&#233;ographie</Text>
+			<Gender>Female</Gender>
+			<Plural>0</Plural>
+		</French>
 		<German>Geography</German>
 		<Italian>Geography</Italian>
 		<Spanish>Geography</Spanish>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_TECH_GEOGRAPHY_QUOTE</Tag>
 		<English>"Leave no stone unturned."[PARAGRAPH:2]- Euripides, Heraclidae</English>
-		<French>"Leave no stone unturned."[PARAGRAPH:2]- Euripides, Heraclidae</French>
+		<French>"La g&#233;ographie embrasse la totalit&#233; des choses, de m&#234;me que l'image d'une t&#234;te la repr&#233;sente tout enti&#232;re."[PARAGRAPH:2]- Ptol&#233;m&#233;e, G&#233;ographie</French>
```

Comment:
That's a good quote, but I would like to keep the actual quotes consistent between translations. Are the precedents of different quotes in the base game?

Maybe this one should feed back into the English text anyway.

Reply:
In this case, I looked up the actual French translation of the Heraclidae, and found that it didn't work at all—the idiom is translated into something different, as idioms tend to do, and unrelated to geography. So I tried to find something else but also ancient Greek. Let me know what you want to do in this case (I can change the English to Ptolemy too).

Reply:
Yes, let's use the Ptolemy quote in English too.